LESSONS.md

```diff
@@ -0,0 +1,95 @@
+# Part 1
+After switching to my branch you need to run `vagrant reload`. This will force vagrant to sync the outside folder with the folder inside vagrant.
+
+```
+New Directory Structure
+-app
+-database
+  schema.md
+  schema.sql
+-public
+  /css
+  /js
+  index.php
+-resources
+```
+
+Because of my directory changes we will need to update apache a bit.
+
+```bash
+vagrant ssh
+cd /etc/apache2/sites-enabled
+sudo nano 000-default.conf
+# Change `DocumentRoot /var/www/html` to `DocumentRoot /var/www/html/public`
+# Control+X to save
+# Yes, save what is in the buffer
+# Yes to the file name
```

Comment:
after demonstrating the reason for making changes to apache, we can keep configuration files in source control and symlink them in our Vagrantfile. That way we don't have to make manual changes every time we `vagrant up`.

Reply:
I'm going to redo this so we don't have to make changes, doesn't make sense to do this... yet.. ;)